app/dao/__init__.py

```diff
@@ -1,6 +1,55 @@
 from sqlalchemy.exc import SQLAlchemyError
+from werkzeug.datastructures import MultiDict
+from sqlalchemy.orm.relationships import RelationshipProperty
+from app import db
 
 
 # Should I use SQLAlchemyError?
 class DAOException(SQLAlchemyError):
     pass
+
+
+class DAOClass(object):
```

Comment:
Thought we talked about not doing this sort of stuff now, pushing queries into the DAO, as explicit queries, until we understood the bounds of our requirements better?